pkg/polaris/patcher.go

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (C) 2019 Synopsys, Inc.
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package polaris
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	appsv1 "k8s.io/api/apps/v1"
+	appsv1beta1 "k8s.io/api/apps/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func fromYaml(content string, polaris Polaris) (map[string]runtime.Object, error) {
+	// Basic
+	content = strings.ReplaceAll(content, "${NAMESPACE}", polaris.Namespace)
+	content = strings.ReplaceAll(content, "$(JAEGER_NAMESPACE)", polaris.Namespace)
+	content = strings.ReplaceAll(content, "$(EVENT_STORE_NAMESPACE)", polaris.Namespace)
+	content = strings.ReplaceAll(content, "$(LEADER_ELECTOR_NAMESPACE)", polaris.Namespace)
```

Comment:
why are these different?

Reply:
Those are the currently placeholders in the yaml. Not sure who added them but there is probably a reason. We can change this later if needed.

Reply:
:-)